test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -693,31 +691,23 @@ public class HttpConnectionDispatcherTests
             values["id"] = state.Connection.ConnectionId;
             var qs = new QueryCollection(values);
             context.Request.Query = qs;
-            context.Response.Body = new MemoryStream();
-            var authFeature = new HttpAuthenticationFeature();
-            authFeature.Handler = new TestAuthenticationHandler(context);
-            context.Features.Set<IHttpAuthenticationFeature>(authFeature);
 
             var builder = new SocketBuilder(sp);
             builder.UseEndPoint<TestEndPoint>();
             var app = builder.Build();
             var options = new HttpSocketOptions();
             options.AuthorizationPolicyNames.Add("test");
 
-            // "authorize" user
-            context.User = new ClaimsPrincipal(new ClaimsIdentity(new[] { new Claim(ClaimTypes.NameIdentifier, "name") }));
-
-            var endPointTask = dispatcher.ExecuteAsync(context, options, app);
-            await state.Connection.Transport.Output.WriteAsync(new Message(Encoding.UTF8.GetBytes("Hello, World"), MessageType.Text)).OrTimeout();
+            context.User = new ClaimsPrincipal(new ClaimsIdentity("authenticated"));
 
-            await endPointTask.OrTimeout();
+            // would hang if EndPoint was running
+            await dispatcher.ExecuteAsync(context, options, app).OrTimeout();
 
-            Assert.Equal(StatusCodes.Status200OK, context.Response.StatusCode);
-            Assert.Equal("T12:T:Hello, World;", GetContentAsString(context.Response.Body));
+            Assert.Equal(StatusCodes.Status403Forbidden, context.Response.StatusCode);
         }
 
         [Fact]
-        public async Task AllPoliciesRequiredForAuthorizedEndPoint()
```

Comment:
Oops, I used it as the basis for the new forbid test, but must have deleted the original accidentally when I did the conversion to the new auth 2.0 stuff.